Resources/public/js/views/ez-usermenuview.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Menu View class
+     *
+     * @module ez-usermenuview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_HIDDEN = 'is-user-menu-hidden',
+        CLASS_USER_PROFILE = 'ez-user-profile',
+        CLASS_USER_MENU = 'ez-user-menu',
+        SELECTOR_USER_PROFILE = '.' + CLASS_USER_PROFILE;
+
+    /**
+     * The User Menu view
+     *
+     * @namespace eZ
+     * @class UserMenuView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.UserMenuView = Y.Base.create('userMenuView', Y.View, [], {
+        _clickOutsideUserMenuHandler: null,
+
+        initializer: function () {
+            this.containerTemplate = '<ul class="ez-view-usermenuview"/>';
+
+            this.on('displayedChange', this._toggleUserMenu);
+            this.after('displayedChange', this._setClickOutsideUserMenuHandler);
+        },
+
+        /**
+         * Renders the user menu view
+         *
+         * @method render
+         * @return {eZ.UserMenuView} the view itself
+         */
+        render: function () {
+            this.get('container').addClass(CLASS_HIDDEN).addClass(CLASS_USER_MENU);
```

Comment:
I don't really see the point of the class `ez-user-menu` (`CLASS_USER_MENU`) since we already have `ez-view-usermenuview` on the container.

Also you should not add `is-user-menu-hidden` (`CLASS_HIDDEN`) here because the `displayed` attribute might be `true` and also because we should leave this responsibility to the `displayedChange` handler. So to clean up this, you should:

1. add this class to the `containerTemplate` in `initializer`
1. set the `displayed` attribute as read only so that the `toggleDisplayed` method is the only way to change it (that also means you have to use `_set` in that method).